content/applications/sales/sales/products_prices/products/product_images.rst

```diff
@@ -1,130 +1,168 @@
-their barcode, keeping your focus on what matters in your business.
+Having appropriate product images in Odoo is useful for a number of reasons. However, if a lot of
+products need images, it can become incredibly time-consuming.
+
+But, with *Google Custom Search* API configured within the Odoo database, finding product images for
+products (based on their barcode) is extremely efficient.
 
 .. _product_images/configuration:
 
 Configuration
 =============
 
-This functionnality requires configuration both on Google and on Odoo.
+In order to utilize *Google Custom Search* within an Odoo database, configuration **must** be
+completed on both -- Google and Odoo.
 
-With a free Google account, you can get up to 100 free images per day. If you need a higher rate,
-you'll have to upgrade to a billing account.
+.. note::
+   Free Google accounts allow up to 100 free images per day. If a higher amount is needed, a billing
+   upgrade is required.
 
 .. _product_images/google-api-dashboard:
 
 Google API dashboard
 --------------------
 
 #. Go to the `Google Cloud Platform API & Services <https://console.developers.google.com/>`_ page
-   to generate Google Custom Search API credentials. Log in with your Google account.
+   to generate Google Custom Search API credentials. Then, log in with a Google account. Next, agree
+   to their :guilabel:`Terms of Service` by checking the box, and clicking :guilabel:`Agree and
+   Continue`.
 
-#. Select or create an API project to store the credentials. Give it an explicit name
-   (e.g. Odoo Images).
+#. Select (or create) an API project to store the credentials. Give it a memorable
+   :guilabel:`Project Name`, select a :guilabel:`Location` (if any), then click :guilabel:`Create`.
 
-#. In the credentials section, click on **Create Credentials** and select **API Keys**.
+#. With the :guilabel:`Credentials` selected in the left sidebar. Click :guilabel:`Create
+   Credentials`, and selecting :guilabel:`API key` from the drop-down menu.
 
-   .. image:: product_images/gcp-api-services.png
+   .. image:: product_images/credentials-api-key.png
       :align: center
-      :alt: API & Services page on Google Cloud Platform
+      :alt: API & Services page on Google Cloud Platform.
+
+#. Doing so reveals an :guilabel:`API key created` pop-up window, containing a custom :guilabel:`API
+   key`. Copy and save :guilabel:`Your API key` in the pop-up window -- it will be used later. Once
+   the key is copied (and saved for later use), click :guilabel:`Close` to remove the pop-up window.
 
-#. Save your **API Key**. You'll need it for the next step in Odoo!
+   .. image:: product_images/api-key-pop-up.png
+      :align: center
+      :alt: The API key created pop-up window that appears.
 
-#. Use the search bar to look for **Google Custom Search API** and select it.
+#. Use the search bar on the page, and search for `Custom Search API`, and select it.
```

Comment:
We're supposed to use GUI labels for search bars. For the text, I just use whatever it says in the search bar before you click on it (ex. "Search...").